src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -230,24 +237,48 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
                 label={t('registration.files_and_license.conditions_for_using_file')}
                 required
                 onChange={({ target: { value } }) => setFieldValue(field.name, value)}>
-                {licenses
-                  .filter((license) => license.version === 4 || !license.version)
-                  .map((license) => (
-                    <MenuItem
-                      data-testid={dataTestId.registrationWizard.files.licenseItem}
-                      key={license.id}
-                      value={license.id}
-                      divider
-                      dense
-                      sx={{ gap: '1rem' }}>
-                      <ListItemIcon>
-                        <img style={{ width: '5rem' }} src={license.logo} alt={license.name} />
-                      </ListItemIcon>
-                      <ListItemText>
-                        <Typography>{license.name}</Typography>
-                      </ListItemText>
-                    </MenuItem>
-                  ))}
+                {activeLicenses.map((license) => (
+                  <MenuItem
+                    data-testid={dataTestId.registrationWizard.files.licenseItem}
+                    key={license.id}
+                    value={license.id}
+                    divider
+                    dense
+                    sx={{ gap: '1rem' }}>
+                    <ListItemIcon>
+                      <img style={{ width: '5rem' }} src={license.logo} alt={license.name} />
+                    </ListItemIcon>
+                    <ListItemText>
+                      <Typography>{license.name}</Typography>
+                    </ListItemText>
+                  </MenuItem>
+                ))}
+                <MenuItem
+                  data-testid={dataTestId.registrationWizard.files.licenseItemShowOlderVersion}
+                  title={t('registration.files_and_license.show_all_older_versions')}
+                  sx={{ display: 'flex', justifyContent: 'center' }}
+                  onClickCapture={(e) => {
+                    e.stopPropagation();
```

Comment:
er denne for å unngå at den lukkes ved trykk?

Reply:
Jepp. Vanlig onClick fanger ikke boblingen, men onClickCapture gjør det.